src/test/java/org/broadinstitute/hellbender/utils/bigquery/AvroFileReaderUnitTest.java

```diff
@@ -0,0 +1,43 @@
+package org.broadinstitute.hellbender.utils.bigquery;
+
+
+import org.apache.avro.generic.GenericRecord;
+import org.broadinstitute.hellbender.GATKBaseTest;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import org.apache.avro.Schema;
+
+import java.util.*;
+
+/**
+ * A class to test the functionality of {@link AvroFileReader}.
+ */
+public class AvroFileReaderUnitTest extends GATKBaseTest {
+
+    private static final String avroFileName = "src/test/java/org/broadinstitute/hellbender/utils/bigquery/avro_test_avro_test_file.avro";
+    private static final AvroFileReader avroFile = new AvroFileReader(avroFileName);
+
+    @Test()
+    public void testGetSchema() {
+        Schema avroFileSchema = avroFile.getSchema();
+        System.out.println("avroFileSchema");
```

Comment:
are these print lines needed for the test? probably remove them (here and in testAvroFileNext below)